docs/developer-guide/working-with-rules.md

```diff
@@ -261,6 +261,8 @@ Once you have an instance of `SourceCode`, you can use the methods on it to work
 * `getTokensAfter(nodeOrToken, count)` - returns `count` tokens after the given node or token.
 * `getTokensBefore(nodeOrToken, count)` - returns `count` tokens before the given node or token.
 * `getTokensBetween(node1, node2)` - returns the tokens between two nodes.
+* `getTokenOrCommentAfter(nodeOrToken)` - returns the first token or comment after the given node or token.
```

Comment:
I've moved the discussion of how we want to handle these methods to https://github.com/eslint/eslint/issues/7810.

If we can land that first, I can update this PR accordingly and remove these two methods from the docs in this PR.